tiny_dnn/core/backend.h

```diff
@@ -40,118 +18,128 @@ class context;
 
 enum class backend_t { tiny_dnn, nnpack, libdnn, avx, opencl };
 
-inline std::ostream& operator << (std::ostream& os, backend_t type) {
-    switch (type) {
-        case backend_t::tiny_dnn: os << "TinyDNN"; break;
-        case backend_t::nnpack:   os << "NNPACK";  break;
-        case backend_t::libdnn:   os << "LibDNN";  break;
-        case backend_t::avx:      os << "AVX";     break;
-        case backend_t::opencl:   os << "OpenCL";  break;
-        default:
-            throw nn_error("Not supported ostream enum.");
-            break;
-    }
```

Comment:
What do you mean exactly? Leaving the `break` statement on the same line? In this case, see my comment above.